vue-project/src/components/SimulationReports.vue

```diff
@@ -63,7 +63,7 @@ const getPrintJobs = async () => {
 			printJobs.value = await response.json();
 		} else {
 			console.log("Error fetching data");
```

Comment:
I feel like these two `console.log`s could be consolidated to `console.log("Error response from server: " + String(response.status))`